Sources/General/KingfisherError.swift

```diff
@@ -139,6 +140,13 @@ public enum KingfisherError: Error {
         /// - original: The original image data, if exists.
         /// - serializer: The `CacheSerializer` used for the image serializing.
         case cannotSerializeImage(image: KFCrossPlatformImage?, original: Data?, serializer: CacheSerializer)
+
+        /// - cannotCreateCacheFile: Cannot create target cache file at the given path and key. Code 3009.
```

Comment:
Depending on if you intend to reuse this error case elsewhere or not, I would have given an explanation that this is due to the file manager not returning a clear error message.

Reply:
I agree with you!